.gitignore

```diff
@@ -6,4 +6,5 @@
 .project
 .cproject
 /.metadata/
+/plugins*
```

Comment:
Drop this from the PR, please — you can add local, non-shared exclusions to the `.git/info/exclude` file in your clone, that way they're separate from the shared repo files. (I only just recently discovered that in the `gitignore` manpage myself.)

Reply:
Done